app/views/catalog/_show_documents.html.erb

```diff
@@ -63,8 +32,45 @@
 <script type="text/javascript">
 $(function() {
 
-  // Wire DataTable for the file list.
-  $('#files-table').DataTable();
+  $('#files-table').DataTable({
```

Comment:
We stole most of the DataTables configuration from PDC Describe: https://github.com/pulibrary/pdc_describe/blob/main/app/views/works/_s3_resources.html.erb#L74